### PR TITLE
<InputWithOptions/> - fix unable to focus editable option

### PR DESCRIPTION
**Pull request by gendeld**
First reported [here](https://wix.slack.com/archives/C39FTGUBZ/p1612082350159600).
Unable to focus inputs inside `InputWithOptions`'s dropdown.

Before:
https://www.wix-style-react.com/?path=/story/introduction-playground--playground&snippet=3e51cd5b-870a-4c56-be16-542a1c046397

After:
https://preview.wix-style-react.com/pr_6670/?path=/story/introduction-playground--playground&snippet=3e51cd5b-870a-4c56-be16-542a1c046397

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6670

File: packages/wix-style-react/src/InputWithOptions/test/InputWithOptions.e2e.js [hunk 1]
```diff
@@ -9,7 +12,7 @@ import {
   testStories,
 } from '../docs/storySettings';
 
-describe('InputWithOptions', () => {
+fdescribe('InputWithOptions', () => {
```
Comment (shlomitc) [inline packages/wix-style-react/src/InputWithOptions/test/InputWithOptions.e2e.js]:
```suggestion
describe('InputWithOptions', () => {
```

File: packages/wix-style-react/src/ListItemEditable/ListItemEditable.uni.driver.d.ts [hunk 2]
```diff
@@ -4,7 +4,9 @@ import { StatusIndications } from '../common';
 export interface ListItemEditableUniDriver extends BaseUniDriver {
   getPlaceholder(): Promise<string>;
   isApproveButtonDisabled(): Promise<boolean>;
+  isInputFocused(): Promise<boolean>;
   enterText(text: string): Promise<void>;
+  clickInput(): Promise<void>;
```
Comment (shlomitc) [inline packages/wix-style-react/src/ListItemEditable/ListItemEditable.uni.driver.d.ts:9]: Isn't focus is part of `enterText?`

File: packages/wix-style-react/src/ListItemEditable/ListItemEditable.uni.driver.d.ts [hunk 3]
```diff
@@ -4,7 +4,9 @@ import { StatusIndications } from '../common';
 export interface ListItemEditableUniDriver extends BaseUniDriver {
   getPlaceholder(): Promise<string>;
   isApproveButtonDisabled(): Promise<boolean>;
+  isInputFocused(): Promise<boolean>;
```
Comment (shlomitc) [inline packages/wix-style-react/src/ListItemEditable/ListItemEditable.uni.driver.d.ts:7]: could be private

File: packages/wix-style-react/src/ListItemEditable/ListItemEditable.uni.driver.d.ts (hunk 2 above)
Comment (gendeld) [reply packages/wix-style-react/src/ListItemEditable/ListItemEditable.uni.driver.d.ts:9]: I'm testing specifically the focusing ability

Comment (shlomitc) [reply packages/wix-style-react/src/ListItemEditable/ListItemEditable.uni.driver.d.ts:8]: So it can sit in the private driver

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6670/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6670

**Pull request merged**